btrace-core/src/test/java/org/openjdk/btrace/core/BTraceUtilsTest.java

```diff
@@ -0,0 +1,20 @@
+package org.openjdk.btrace.core;
+
+import org.junit.jupiter.api.Test;
+
+
+class BTraceUtilsTest {
+
+    static class SuperClass {
+        int a;
+    }
+
+    static class Subclass extends SuperClass {
+
+    }
+    @Test
+    public void getIntTest() {
+        Subclass a = new Subclass();
+        assert BTraceUtils.Reflective.getInt("a", a) == 0;
+    }
```

Comment:
Please, add more cases - eg. trying to access a completely non-existing field, having class hierarchy more than 2 levels deep etc.